mopidy/internal/path.py

```diff
@@ -196,23 +196,23 @@ def find_mtimes(root, follow=False):
     return mtimes, errors
 
 
-def check_file_path_is_inside_base_dir(file_path, base_path):
-    assert not file_path.endswith(os.sep), (
-        'File path %s cannot end with a path separator' % file_path)
-
+def is_local_path_inside_base_dir(local_path, base_path):
```

Comment:
``path`` is good as long as we don't use
``from os import path`` which we wouldn't ;-)

BTW
why do we check on ``local_path.endswith(os.sep)`` ?

I hope i can do the remaining changes tonight.